src/extensions/anvil/algorithm.ts

```diff
@@ -560,10 +559,9 @@ namespace HphobHphil {
             }
         }
         return { hphob, hphil };
-    }
-
-    const testPoint = v3zero();
```

Comment:
I would create that at the module/file level for reuse, not as part of the object, so it is not accessible elsewhere